pkg/vdr/web/did.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package web
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+)
+
+const (
+	defaultPath  = "/.well-known/doc.json"
+	documentPath = "/doc.json"
+)
+
+// parseDIDWeb consumes a did:web identifier and returns the URL location of the did Doc.
+func parseDIDWeb(id string) (string, error) {
+	var urlString string
+
+	// parses did to make sure it conforms to generic did standard
+	parsedDID, err := did.Parse(id)
+	if err != nil {
+		return urlString, fmt.Errorf("invalid did, does not conform to generic did standard --> %w", err)
+	}
+
+	if parsedDID.Method != "web" {
```

Comment:
How is this even possible if your vdr is meant for "web" method?  It will not get invoked unless it is "web".

Reply:
I've made the changes you mentioned, I guess I was still a little unclear on the role of vdr in aries so I'll read into it more and keep that in mind for future changes : )